src/rspec/spec/teacher_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'person_spec'
+require_relative '../files'
+
+class Teacher < Person
+  attr_accessor :specialization, :age, :name
+
+  def initialization(age, name, specialization)
+    super(age, name)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```

Comment:
- [ ] Instead of redefining the class again, kindly use require_relative to make it available in this file 👍
- [ ] Please ensure to test the `can_use_services?` method 👍

Reply:
Noted and edited. Thanks for the feedback